src/docs/tool-xray.dox

```diff
@@ -80,45 +103,23 @@ XRay log and the \c wtperf binary that we used to generate it.
 $ cp <wtperf> <xray_log> .
 @endcode
 
-## Step 2: Install LLVM
-
-The \c xray_to_optrack utility requires LLVM in order to parse the XRay log
-file. So we'll need to install LLVM via our package manager.
-
-@code
-$ sudo apt install llvm
-@endcode
-
-LLVM needs to be version 8 or higher. Check the version like this:
-
-@code
-$ llvm-config -version
-@endcode
-
-If your distribution's default \c llvm-config isn't from the 8 series, you'll
-need to move one with a major version of 8 into the \c $PATH such that it gets
-invoked instead.
-
-The \c llvm-config command line tool is used to supply the required compiler
-and linker flags to build programs such like \c xray_to_optrack on top of LLVM.
-
-## Step 3: Configure and build WiredTiger with LLVM flags
+## Step 2: Configure and build WiredTiger with LLVM flags
 
 Supply the \c -DENABLE_LLVM flag to your configuration to build anything with a
 dependency to LLVM.
```

Comment:
Can you add a note here that `xtray_to_optrack` requires both `-DENABLE_LLVM=1` and a clang toolchain, otherwise it won't be compiled?

Reply:
The llvm installation (in xray instrumentation step 1) should have installed the clang toolchain to enable the build. I made the suggested update in [549be32](https://github.com/wiredtiger/wiredtiger/pull/8198/commits/549be3261dcbe87758adda12621c54cc407689e3)